pkg/client/simple/vfsclientset/clientset.go

```diff
@@ -161,6 +161,9 @@ func DeleteAllClusterState(basePath vfs.Path) error {
 		if strings.HasPrefix(relativePath, "instancegroup/") {
 			continue
 		}
+		if strings.HasPrefix(relativePath, "igconfig/") {
```

Comment:
It's the config specific to particular instancegroups. First I partition by IG role (because that's the granularity our IAM roles currently have), then by IG. Whether the config is "full" or versioned is secondary to this partitioning.